debugview/src/test/java/com/revenuecat/purchases/debugview/DebugViewSnapshotTest.kt

```diff
@@ -0,0 +1,173 @@
+package com.revenuecat.purchases.debugview
+
+import android.app.Activity
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import app.cash.paparazzi.Paparazzi
+import com.android.ide.common.rendering.api.SessionParams
+import com.revenuecat.purchases.Package
+import com.revenuecat.purchases.debugview.models.SettingGroupState
+import com.revenuecat.purchases.debugview.models.SettingScreenState
+import com.revenuecat.purchases.debugview.models.SettingState
+import com.revenuecat.purchases.debugview.models.testOffering
+import com.revenuecat.purchases.models.StoreProduct
+import com.revenuecat.purchases.models.SubscriptionOption
+import io.mockk.mockk
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@Suppress("TooManyFunctions")
+class DebugViewSnapshotTest {
+
+    private val maxHeight = 3000.dp
+
+    @get:Rule
+    val paparazzi = Paparazzi(
+        renderingMode = SessionParams.RenderingMode.V_SCROLL,
+    )
+
+    private lateinit var notConfiguredStateFlow: MutableStateFlow<SettingScreenState>
+    private lateinit var notConfiguredViewModel: DebugRevenueCatViewModel
+    private lateinit var configuredStateFlow: MutableStateFlow<SettingScreenState>
+    private lateinit var configuredViewModel: DebugRevenueCatViewModel
+
+    private var toastDisplayedCallCount = 0
+    private var purchasePackageCallCount = 0
+    private var purchaseProductCallCount = 0
+    private var purchaseOptionCallCount = 0
+
+    @Before
+    fun setUp() {
+        toastDisplayedCallCount = 0
+        purchasePackageCallCount = 0
+        purchaseProductCallCount = 0
+        purchaseOptionCallCount = 0
+        setupTestViewModels()
+    }
+
+    @Test
+    fun notConfiguredStateDisplaysCorrectly() {
+        snapshotTest {
+            InternalDebugRevenueCatScreen(
```

Comment:
I tried to use the bottom sheet instead of the `Screen` composable directly, but couldn't get it to work. I changed to this and it worked like a charm, and I think it covers most of what we want.

Reply:
Yeah this is fine 👍🏻